### PR TITLE
add top global graph on issue #26

### PR DESCRIPTION
**Pull request by maulana20**
completing in issue #26

Signed-off-by: maulana <maulana20@versatech.co.id>

### CODE REVIEW

File: src/components/global/DataGraph.vue [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            <content-loader :is-loading="isLoading" @refresh-data="renderChart">
+                <template v-slot:content>
+                    <div style="text-align: left;" class="mb-1">
+                        <b-button @click="changeCategory('positive')" :variant="activeTab === 'positive'? 'success': 'outline-success'" pill>Positif</b-button>
+                        <b-button @click="changeCategory('died')" :variant="activeTab === 'died'? 'success': 'outline-success'" class="ml-2" pill>Meninggal</b-button>
+                        <b-button @click="changeCategory('recovery')" :variant="activeTab === 'recovery'? 'success': 'outline-success'" class="ml-2" pill>Sembuh</b-button>
+                    </div>
+                    <highcharts :options="chartOption"></highcharts>
+                    <div class="mt-1">
+                        <b-button @click="showAllProvince()" variant="outline-primary" pill>{{showTop10? 'Tampilkan semua provinsi': 'Tampilkan 10 provinsi'}}</b-button>
```
Comment (pararang) [inline src/components/global/DataGraph.vue]: The wording of the province in the global section will make the user confuse.

File: src/components/global/DataGraph.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                    chartHeight = 500
+                    categories = categories.slice(0, 10)
+                    dataSeries = dataSeries.slice(0, 10)
+                }
+
+                this.chartOption.chart.height = chartHeight
+                this.chartOption.xAxis.categories = categories
+                this.chartOption.series = [{name: nameSeries, data: dataSeries}]
+            },
+            showAllProvince() {
```
Comment (pararang) [inline src/components/global/DataGraph.vue]: please refactor the function name to `showAllCountry()`

File: src/components/global/DataGraph.vue [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+                this.showTop10 = !this.showTop10
+                this.changeCategory(this.activeTab)
+
+                // Scroll to the top of chart
+                const element = document.getElementById('chart-indonesia')
```
Comment (pararang) [inline src/components/global/DataGraph.vue]: please check this out, make sure to scroll up to the global section, not to Indonesia

Comment (maulana20) [reply src/components/global/DataGraph.vue]: sorry @pararang .. I forget not to review it again, i'll fix it later

**Pull request merged**